app/src/main/res/layout/view_page_action_overflow.xml

```diff
@@ -19,7 +19,7 @@
 
             <LinearLayout
                 android:id="@+id/overflowList"
-                android:layout_width="wrap_content"
+                android:layout_width="match_parent"
```

Comment:
This will prevent the items from being wider than the "Customize toolbar" item, since the "Customize toolbar" item is now the only one that has `wrap_content`. (And there are some languages where certain items are longer than "Customize toolbar", and will be cut off.)

The proper solution might be to leave these items as `wrap_content`, and then measure the width of the widest item at runtime, and adjust the other widths to match the maximum.

Reply:
Tried to get the max width from the list items and adjust them by setting either `view.width` or `layoutParams.width`, but it will create a weird space at the bottom of the list 🤔 

I have another way to add menu items inside one single `LinearLayout` but the `border` view makes the entire view wider... any suggestions for this?